debian/control

```diff
@@ -343,8 +343,9 @@ Description: kubernetes events module for ceph-mgr
 Package: ceph-mgr-cephadm
 Architecture: all
 Depends: ceph-mgr (= ${binary:Version}),
-	 cephadm,
-	 python3-six,
+         cephadm,
+         python3-six,
+         python3-openssl
```

Comment:
note, @tchaikov just added a PR that moves the core modules to a new package. Might need a rebase.

Reply:
Yeah, this hunk should move to ceph-mgr-modules-core